ClosedXML/Excel/ConditionalFormats/XLConditionalFormat.cs

```diff
@@ -222,6 +212,19 @@ public IXLConditionalFormat SetStopIfTrue(bool value)
             return this;
         }
 
+        public IXLConditionalFormat CopyTo(IXLWorksheet targetSheet)
+        {
+            var targetRange = targetSheet.Range(Range.RangeAddress.WithoutWorksheet());
```

Comment:
Do you think that all `CopyTo` methods should first check that `targetSheet` is different to the "current sheet"? For example, it doesn't make sense to copy a conditional format from a sheet into the same sheet.

Reply:
True. Added a validation and two more unit tests.